vllm/platforms/cuda.py

```diff
@@ -195,6 +195,10 @@ def get_attn_backend_cls(cls, selected_backend, head_size, dtype,
                 logger.info("Using Triton MLA backend.")
                 return "vllm.attention.backends.triton_mla.TritonMLABackend"
         if use_v1:
+            if os.environ.get("VLLM_V1_USE_TRITON_BACKEND", "0") == "1":
+                logger.info_once("Using ROCm Attention backend on V1 engine.")
```

Comment:
Nit: This may be controversial but I think that using "Triton" instead of "ROCm" in this log is better for the user.

Reply:
Changed it to `Triton/ROCm Attention`. Note that there is also [this](https://github.com/vllm-project/vllm/pull/14071) relevant PR.